DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: admiral
 Type: Package
 Title: ADaM in R Asset Library
-Version: 0.12.2
+Version: 0.12.2.9000
```

Comment:
```suggestion
Version: 0.12.3
```

since this should be a one-time fix, no need to add development flow to it?

Reply:
I don't think we should do a another bug fix release.  should just ship out with `1.0`

Reply:
im unclear if CRAN would even accept a third patch in 3 weeks

Reply:
If so, we can remove my suggestion, but merge to `main` instead of `patch` right?

Reply:
yes merge to `main`

Reply:
@rossfarrugia , would like to have this fixed soon because the issue was detected by a Roche study using admiral.

Maybe we could do a patch release but do not submit it to CRAN? However, it may be confusing for health authorities if a study does a submission with admiral 0.12.3 and this version is not available on CRAN.

What do you think?

Reply:
my take on this is its not a minor bug in terms of impact - its a common derivation that pretty much every study will use, it relates to death information which is key safety info, and currently it doesn't work as documented in admiral. so for me thats something i'd not want to wait for as a user. so i'd be in favour of releasing as 0.12.3 and if CRAN don't accept then nothing we can do about that. at least users would have an option to install and use this from github until 1.0.0 comes later.

Reply:
okay 

@bundfussr  is this branched off release tag 12.2?

Reply:
I branched off `patch`, which should be the same as tag `12.2`.

Reply:
Updated to 0.12.3.